server/service/src/sync/file_synchroniser.rs

```diff
@@ -11,40 +11,35 @@ use repository::{
     RepositoryError,
 };
 
-use crate::static_files::StaticFile;
+use crate::static_files::{StaticFile, StaticFileCategory};
```

Comment:
This file is refactored to use error similar to existing sync API.

Also added early return to try help with readability